6_visualize/src/combine_animation_frames.R

```diff
@@ -33,24 +79,28 @@ combine_animation_frames <- function(gif_file, animation_cfg, frame_ind_intro=NU
   outro_delay <- 200
   final_delay <- 700
   freeze_delay <- 150
-  # **trash code for now:
+
   calc_delays <- function(delay, start_frame, end_frame){
     paste(paste(sprintf('-d%s "#', delay), seq(start_frame-1, end_frame-1), sep = '') %>%
             paste('"', sep = ''), collapse = " ")
   }
-  intro_delays <- calc_delays(intro_delay, 1, n_intro)
-  storm_delays <- calc_delays(storm_delay, n_intro+1, total_frames-n_outro-1)
+
+  intro_delay_str <- calc_delays(intro_delay, 1, n_intro)
+  storm_delay_str <- calc_delays(storm_delay, n_intro+1, total_frames-n_outro-1)
   # freeze the last storm frame too for as long as we are showing each outro frame:
-  last_storm_delay <- calc_delays(freeze_delay, total_frames-n_outro, total_frames-n_outro)
-  outro_delays <- calc_delays(outro_delay, total_frames-n_outro+1, total_frames-1)
-  final_delay <- calc_delays(final_delay, total_frames, total_frames)
+  last_storm_delay_str <- calc_delays(freeze_delay, total_frames-n_outro, total_frames-n_outro)
+  outro_delay_str <- calc_delays(outro_delay, total_frames-n_outro+1, total_frames-1)
+  final_delay_str <- calc_delays(final_delay, total_frames, total_frames)
 
-  gifsicle_command <- sprintf('gifsicle -b -O3 %s %s %s %s %s %s --colors 256', gif_file, intro_delays, storm_delays, last_storm_delay, outro_delays, final_delay)
+  gifsicle_command <- sprintf('gifsicle -b -O3 %s %s %s %s %s %s --colors 256', gif_file,
```

Comment:
Do you know any reference that explains gifsicle commands?

Reply:
I haven't been able to

Reply:
https://www.lcdf.org/gifsicle/man.html useful?

Reply:
^ that and stackoverflow are where I find most of my gifsicle help!